content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+<time_off/request-allocation>` appears form appears.
+
+.. image:: time_off/dashboard.png
+   :align: center
+   :alt: Time off dashboard view with the legend, time off summaries, and view buttons highlighted.
+
+My time off
+-----------
+
+To view a list of all the user's time off requests, navigate to :menuselection:`Time Off application
+--> My Time Off --> My Time Off`. All time off requests appear in a list view, both past and
+present. Other than the employee's name, the list includes the following information for each
+request: the :guilabel:`Time Off Type`, :guilabel:`Description`, :guilabel:`Start Date`,
+:guilabel:`End Date`, :guilabel:`Duration`, and the :guilabel:`Status`. A new time off request can
+be made from this view. Click the :guilabel:`Create` button to :ref:`request time off
+<time_off/request-time-off>`.
+
+My allocations
+--------------
+
+To view a list of all the users allocations, navigate to :menuselection:`Time Off application -->
+My Time Off --> My Allocations`. All allocations and requested allocations appear in a list view.
+The information presented includes: the :guilabel:`Time Off Type`, :guilabel:`Description`,
+:guilabel:`Duration`, :guilabel:`Allocation Type`, and the :guilabel:`Status`. A new allocation
+request can be made from this view. Click the :guilabel:`Create` button to :ref:`request an
+allocation <time_off/request-allocation>`.
+
+Overview
+========
+
+To view a color-coded schedule of both the user's time off, and/or the team managed by them,
+navigate to :menuselection:`Time Off application --> Overview`. This presents a calendar with the
+default filter of :guilabel:`My Team`, in a month view. To change the time period displayed, click
+on either the :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, or :guilabel:`Year` buttons to
+present the calendar in that corresponding view.
+
+Each team member is displayed on a line, and any time off they requested, regardless of the status
+(:guilabel:`Validated` or :guilabel:`To Approve`), appears on the calendar. Each employee is
+color-coded. The employee's color is selected at random and does not correspond to the type of time
+off they requested. The status of the time of is represented by the color of the request either
+appearing solid (:guilabel:`Validated`) or striped (:guilabel:`To Approve`). The number of days or
+hours requested is written on the request (if there is enough space). At the bottom of the calendar,
+a bar graph shows how many people are projected to be out on any given day. The number on the bar
+represents the number of employees out for those highlighted days.
+
+Hover over a time off entry to view the details for the specific time off entry. The total number of
+hours or days are listed, along with the start and end time of the time off.
+
+.. image:: time_off/overview.png
+   :align: center
+   :alt: Overview of the user's team, with time off requests shown.
+
```

Comment:
Suggestion: I think that the hierarchy of content in this doc warrants this _Overview_ section to be above the _Allocate time off_, _Request time off_, _Request allocation_, _Approvals_, _My time off_ sections since these sections are all H2s.